migrations/schema.sql

```diff
@@ -754,35 +735,6 @@ CREATE TABLE auth.users (
 COMMENT ON TABLE auth.users IS 'Auth: Stores user login data within a secure schema.';
 
 
---
--- Name: decrypted_key; Type: VIEW; Schema: pgsodium; Owner: -
---
-
-CREATE VIEW pgsodium.decrypted_key AS
```

Comment:
@michelp do you know why this got dropped now? If I recreate `pgsodium` it gets dumped again.

Reply:
I think I got it - this was fixed in pgsodium 3.1.1 but now [this issue](https://github.com/supabase/postgres/blob/00837a237134dc0cb7ba3d48fa96b67cc55da2a3/migrations/db/migrations/20221207154255_create_pgsodium_and_vault.sql#L13-L17) is back, so we need to revert the migration.